### ISSUE TITLE
Publishing on greasyfork.org, userstyles.world, and reddit

### ISSUE THREAD

**Issue by rybak** [rybak/atlassian-tweaks#4]
I've published two of the scripts to https://greasyfork.org:

- https://greasyfork.org/en/scripts/456690-bitbucket-commit-links-in-prs
- https://greasyfork.org/en/scripts/456692-jira-pull-request-link-improver

I also wanted to publish the scripts to https://userstyles.world/, but they are [having issues with login via OAuth](https://user-images.githubusercontent.com/624072/209155431-320bc6a7-c51f-48a6-8d27-a2e1c889f801.png) at the moment: https://github.com/userstyles-world/userstyles.world/issues/134#issuecomment-1339587753

There are also [r/bitbucket](https://www.reddit.com/r/bitbucket/) and [r/jira](https://www.reddit.com/r/jira/) on reddit.

Should we publicize the user scripts and user styles on reddit? @dtruebin, @svlukas what do you think?

P.S. While looking at other scripts on greasyfork, I noticed that people often confuse Jira/Bitbucket Cloud with Jira/Bitbucket Server. Most of the scripts are historically for the Server variant (as are scripts and styles in [atlassian-tweaks](https://github.com/rybak/atlassian-tweaks)) and they are usually not compatible with the Cloud variant.

**Comment by dtruebin** [comment]
Why not 🧐

Please consider adding a screenshot to https://greasyfork.org/en/scripts/456692-jira-pull-request-link-improver.
By the way, I am pretty sure Jira/Bitbucket Cloud have a free tier, so the scripts can be checked for compatibility with them.

**Comment by rybak** [comment]
> Please consider adding a screenshot to https://greasyfork.org/en/scripts/456692-jira-pull-request-link-improver.

Good idea, done. Also added on in the README: c1712ff3f57d0b35ebff9c3c45b45f07c2e2a008.

**Comment by rybak** [comment]
* Created a script set on greasyfork.org: https://greasyfork.org/en/scripts?set=585074
  * It includes the newly published https://greasyfork.org/en/scripts/457217-jira-project-icon-as-tab-icon as well
* Posted to reddit:
  * r/atlassian: https://www.reddit.com/r/atlassian/comments/zwizky/atlassian_tweaks_a_collection_of_user_scripts_and/
  * r/jira: https://www.reddit.com/r/jira/comments/zwjfki/atlassian_tweaks_a_collection_of_user_scripts_and/
  * both r/bitbucket and r/confluence are restricted in who can post, though.

**Comment by rybak** [comment]
As it turns out, greasyfork.org also supports publishing user styles: https://greasyfork.org/en/scripts?language=css

- [ ] publish the rest of scripts on greasyfork.org
- [ ] publish CSS on greasyfork.org
- [ ] wait for OAuth issue to be resolved and publish styles to userstyles.world

**Comment by rybak** [comment]
I've found out that [jira_copy_summary.user.js](https://github.com/rybak/atlassian-tweaks/raw/main/jira_copy_summary.user.js) in its current implementation is disallowed to be published on Greasy Fork. It is automatically rejected with error message

> Code uses an unapproved external script: `@require https://raw.githubusercontent.com/odyniec/MonkeyConfig/0eaeb525/monkeyconfig.js`

because of

```
// @require      https://raw.githubusercontent.com/odyniec/MonkeyConfig/0eaeb525/monkeyconfig.js
```

**Comment by rybak** [comment]
Stumbled upon another website for user scripts: https://openuserjs.org. Searched for "Jira" just out of curiosity and on the first page of search results got four Copy Summary analogues:

- https://openuserjs.org/scripts/floodmeadows/Jira_add_descriptive_link
- https://openuserjs.org/scripts/wmosakowskicraft/Jira_Task_to_Clipboard
- https://openuserjs.org/scripts/milanfarkas/Jira_Agile_Task_to_Clipboard
- https://openuserjs.org/scripts/milanfarkas/Jira_Cloud_Copy_Task_Key_and_Title_to_Clipboard

**Comment by rybak** [comment]
https://userstyles.world/ is available again! 😃 

Started off with https://userstyles.world/style/9186/bitbucket-fast-and-furious

I'll import all user styles soon™.

**Comment by rybak** [comment]
Uploaded all userstyles that I wanted to upload to userstyles.world:

- [Bitbucket: header color](https://userstyles.world/style/9187/bitbucket-header-color)
- [Bitbucket: fix breadcrumbs selection](https://userstyles.world/style/9295/bitbucket-fix-breadcrumbs-selection)
- [Bitbucket: hide banner](https://userstyles.world/style/9296/hide-bitbucket-banner-at-the-top)
- [Bitbucket: fast and furious](https://userstyles.world/style/9186/bitbucket-fast-and-furious)
- [Confluence: better monospace](https://userstyles.world/style/9298/confluence-better-monospace)
- [Confluence: hide banner](https://userstyles.world/style/9299/hide-confluence-banner-at-the-top)
- [Confluence: Simple Floating ToC](https://userstyles.world/style/9300/confluence-simple-floating-toc)
- [Jira: better inline code formatting](https://userstyles.world/style/9301/jira-better-inline-code-formatting)

The only exception is [Bitbucket: fix commit tooltip](https://github.com/rybak/atlassian-tweaks#bitbucket--fix-commit-tooltip), which doesn't seem to be needed for modern Bitbucket Server. I haven't uploaded it to [Greasy Fork](https://greasyfork.org/en/scripts?language=all&set=585074) either.

**Comment by rybak** [comment]
### Last Greasy Fork update

Uploaded the rest of userstyles (except "Bitbucket: fast and furious") to Greasy Fork: https://greasyfork.org/en/users/951503-andrybak?language=css&set=585074

As per [comment above](https://github.com/rybak/atlassian-tweaks/issues/4#issuecomment-1385738749) I can't upload "Jira copy summary". And I don't want to bother with ["Jira: Copy code blocks"](https://github.com/rybak/atlassian-tweaks/blob/main/jira-copy-code-blocks.user.js), because I don't use it and it isn't that good UX-wise.

**Comment by rybak** [comment]
> Stumbled upon another website for user scripts: https://openuserjs.org.

I've [tried uploading to this website](https://openuserjs.org/scripts/rybak/Subreddit_tab_icons), and I've decided not to upload scripts of Atlassian Tweaks there for now. See also https://tinyurl.com/24sy79e5 :-/

**Comment by rybak** [comment]
> I've found out that [jira_copy_summary.user.js](https://github.com/rybak/atlassian-tweaks/raw/main/jira_copy_summary.user.js) in its current implementation is disallowed to be published on Greasy Fork. It is automatically rejected with error message

The problem has been solved in commit [9eecf2d](https://github.com/rybak/atlassian-tweaks/commit/9eecf2d1161f354b89c30b5994c72931b432a128) (jira_copy_summary: use full SHA1 hash for monkeyconfig.js, 2023-09-21). The script is now published at Greasy Fork: <https://greasyfork.org/en/scripts/475815-jira-copy-summary>
